README.md

```diff
@@ -231,7 +233,38 @@ A style guide for the [HTML Template Language](https://docs.adobe.com/docs/en/ht
      
     <div data-sly-test="${!hasContent}" class="cq-placeholder"></div>
     ```
-    
+
+    Similarly, if a generic expression is used multiple times, define an identifer with `data-sly-set` and re-use it, for performance reasons.
```

Comment:
I think this goes beyond performance reasons. it also reduces cognitive load

Reply:
Thanks, please check the expanded statements.